src/Api/Test/fixtures/clients.yaml

```diff
@@ -0,0 +1,23 @@
+App\Entity\Client:
```

Comment:
What a mess!!!!
This is not source code but configuration, isn't it? Why is it in src/Api/Test?
In fact you already have a /fixtures folder in your project root, shouldn't this be located there?
In fact you also have a '/tests' folder in the project root. Why not ''/tests/fixtures" ?

If the used fixtures library does suggest some best practices related to how to organize files well let's follow them and not fight it but this seems such a mess that I doubt this is the best practice.

Please enlighten me if I'm wrong.

Reply:
@uanabitarte I see that you have moved fixture yaml's to /fixtures, ok. 
You have also moved the PHP code to /test/api and that leads you to namespaces with "api" instead of "Api". If "tests" will be a source code folder folders inside it should be capitalized.

By the way you also have /src/Tests/Controller/DefaultControllerTest.php or something like that.
Don't know if this is just a trial and can be deleted or if we really need/want it, but if we need it shouldn't we move it to "tests"?

Reply:
/src/Tests/Controller/DefaultControllerTest.php is from the legacy code, I do not know if it is really needed